core/src/level/elements/tile/ExitTile.java

```diff
@@ -0,0 +1,32 @@
+package level.elements.tile;
+
+import basiselements.DungeonElement;
+import level.elements.ILevel;
+import level.tools.Coordinate;
+import level.tools.DesignLabel;
+import level.tools.LevelElement;
+
+public class ExitTile extends Tile {
+
+    /**
+     * Creates a new Tile.
+     *
+     * @param texturePath Path to the texture of the tile.
+     * @param globalPosition Position of the tile in the global system.
+     * @param designLabel Design of the Tile
+     * @param level The level this Tile belongs to
+     */
+    public ExitTile(
+            String texturePath, Coordinate globalPosition, DesignLabel designLabel, ILevel level) {
+        super(texturePath, globalPosition, designLabel, level);
+        this.levelElement = LevelElement.EXIT;
```

Comment:
Ich zieh das jetzt mal nicht weiter durch mit den Suggestions. Aber bitte das `this`in den anderen Klassen auch entfernen, wenn es nicht benötigt wird

```suggestion
        levelElement = LevelElement.EXIT;
```